helm/amko/crds/gdp_def.yaml

```diff
@@ -64,6 +64,8 @@ spec:
                 maximum: 86400
               sitePersistenceRef:
                 type: string
+              pkiProfileRef:
```

Comment:
will this be applied to gslb service only when sitePersistence is set in gslbhostrule? will this take precedence over gslbhostrule pkiProfileRef?

Reply:
We follow the same rules as for persistence profile. GSLB Hostrule always take precedence over GDP 
  1. GSLBHostRule contains site persistence, but it is disabled:
    Regardless of what's in the GDP object, we disable Site Persistence on this GS object so no pki Profile will be set.  2. GSLBHostRule contains site persistence, it is enabled and a pki profile ref is given:
   We enable Site Persistence on the GS object and set the provided ref as the pki ref.
 3. GSLBHostRule doesn't contain PKI, but site persistence is enabled then we inherit the PKi profile from
   the Global filter (GDP object).
   4.GSLBHostrule does not contain site persistence object. We rely on fields in GDP object.